app/views/search/_base.json.jbuilder

```diff
@@ -0,0 +1,14 @@
+json.id result['identifier']
+json.source result['source']
+json.source_link result['source_link']
+json.full_record_link record_url(result['identifier'])
+json.content_type result['content_type']
+json.format result['format']
+json.realtime_holdings_link 'Not Yet Implemented'
+json.publication_date result['publication_date']
+json.title result['title']
+json.links ['Note Yet Implemented']
+json.authors result['creators']
+json.subjects result['subjects']
+json.thumbnail 'Not Yet Implemented'
+json.summary_holdings ['Note Yet Implemented']
```

Comment:
Couple of Not/Note typos here

Reply:
Note yet darn it! ;) Thanks. Will clean them up.